Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md

```diff
@@ -490,6 +490,40 @@ When deploying via AWS, we register the instance to the load balancer and automa
 
 **Note:** Since you more frequently update or replace applications than load balancers, we recommend you reuse existing load balancers in production environments. This will help retain DNS settings that forward traffic to the instance.
 
+#### Deployment Information
+
+Once your instance has been deployed, you can access its information by clicking on it in the instances list view and accessing the instance details page. The right side of the screen displays instance deployment information. _Not all the attributes apply to every instance so some of them might not appear_:
+
+|         Concept      | Description |
+|----------------------|-------------|
+|Support ID            | This ID relates this instance in case you ask for support regarding it. There might be more than one IDs in case this CAM instance contains more than one vendor instances. |
+|Policy                | Cloud Application Manager [Policy Box](../Automating Deployments/deploymentpolicy-box.md) used to deploy this instance. It links directly to the deployment policy box page. |
+|ID                    | Internal instance identifier. |
+|Service ID            | ID for the service this instance uses. |
+|Service               | Type of service included in this instance. It can be an operating system, an application, a script, etc. |
+|Hostname              | Hostname of the instance. |
+|Provider              | [Provider](../Core Concepts/providers.md) in which this instance is deployed. |
+|Provider Instance ID  | AWS instance ids. If the user has enough rights, it shows a link to the resource in the AWS console. |
+|Proxy                 | Proxy of the instance in case it is configured. |
+|Region                | AWS Region in which this instance is deployed. |
+|Availability Zone     | AWS Availability zone|
+|Instance Type         | AWS Instance type. |
+|AMI ID                | AMI ID from the AWS repository. |
+|Network Type          | EC2 Classic or the Virtual private cloud to which this instance belongs to. |
+|Placement group       | _[AWS docs](//docs.aws.amazon.com/AWSEC2/latest/UserGuide/placement-groups.html)_|
+|Load Balancer         | Name of the load balancer that distributes the load across all availability zones in the region. |
+|Autoscaling           | Yes or no. |
+|Elastic Block Storage | Yes or no. |
+|Elastic IP            | Yes or no. |
+|Automatic Backups     | Yes or no. |
```

Comment:
I think this was for RDS instancas no longer supported. It can be removed, right @davisein ?

Reply:
I think you are right, @gsanchezu . I have searched for this property and there is no trace of it in the back-end code. So, yep, I guess this is front-end dead code.